src/action.rs

```diff
@@ -20,8 +20,6 @@ use std::sync::mpsc::Sender;
 ///    2. `Action::Terminate` indicates to `Core` that no new actions should be taken and all
 ///       pending events should be handled.
 ///       After completion `Core` will send `Event::Terminated`.
-// FIXME - See https://maidsafe.atlassian.net/browse/MAID-2026 for info on removing this exclusion.
```

Comment:
Couple of issues with commits:
- Could we keep the history linear by rebasing the change to avoid the merge commit `Merge branch 'fleming' of github.com:maidsafe/routing into fleming`. The first commit seem to be from `https://github.com/maidsafe/routing/pull/1998/files` it probably should not be there?
- Couple of commits have cruft in comment that was not suppose to be there I think: `# # On branch large_enum # Your branch is up-to-date with 'origin/large_enum'. # # Changes to be committed: #	modified: src/authority.rs # # Changes not staged for commit:`
- one commit got a typo: `routng/refactor Remove unnecessary FIXME` -> routing
- `routing/refactor Reword docs for Authority` does not do that but remove the commented out line from previous commit which is strange. should probably squash these 2.

Reply:
Yes, happy enough to do whatever her @jeanphilippeD  but the PR's were a bit confused and I had to work around that and alter git history in these branches. 
If there are git commands you want to be made that is cool, but I am not sure how to do what you are asking here. I know we can squash and merge/rebase etc. in normal github flow, but I don't know how I can do it manually without risking losing commits.

Reply:
Hi @dirvine you should be able to do all of the changes to history using interactive rebase: https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history this page seem pretty decent to describe command line (found searching git rebase interactive). To ensure you do not loose anything, before any history rewrite create a new branch that you won't touch, then this backup branch will always point to the original work.